dataprofiler/tests/labelers/test_column_name_model.py

```diff
@@ -79,12 +79,44 @@ def test_param_validation(self):
             },
         ]
 
-        model = ColumnNameModel(parameters=mock_model_parameters)
+        cls.parameters = {
+            "true_positive_dict": [
+                {"attribute": "ssn", "label": "ssn"},
+                {"attribute": "suffix", "label": "name"},
+                {"attribute": "my_home_address", "label": "address"},
+            ],
+            "false_positive_dict": [
+                {
+                    "attribute": "contract_number",
+                    "label": "ssn",
+                },
+                {
+                    "attribute": "role",
+                    "label": "name",
+                },
+                {
+                    "attribute": "send_address",
+                    "label": "address",
+                },
+            ],
+            "negative_threshold_config": 50,
+            "include_label": True,
+        }
+
+        cls.test_label_mapping = {"ssn": 1, "name": 2, "address": 3}
```

Comment:
Any reason this over:
`model = ColumnNameModel(label_mapping=mock_label_parameters, parameters=mock_model_parameters)`
?

Reply:
would be better ... I'll give it a go 👍